tests/nat/builder/test_builder.py

```
@@ -765,6 +933,416 @@ async def test_built_config():
+
+    config = CustomInstanceConfig()
+    group = FunctionGroup(config=config, instance_name="custom_math_group")
+
+    async def add_func(a: int, b: int) -> int:
+        return a + b
+
+    group.add_function("add", add_func, description="Add two numbers")
+
+    # Function should be stored with simple name internally
+    all_functions = group.get_all_functions()
+    assert "add" in all_functions
```

Ruff (0.12.2)

Use of `assert` detected

(S101)